.github/workflows/docusaurus.yml

```diff
@@ -76,10 +76,3 @@ jobs:
         echo "machine github.com login $GIT_USERNAME password $GITHUB_TOKEN" > ~/.netrc
         # Run docusaurus-publish
         GIT_USER=$GIT_USERNAME npm run publish-gh-pages --prefix website
-    - name: Create comment
```

Comment:
@grovesy - This commit is now #43 as the error building this PR means this commit will never be merged.